torch/package/package_exporter.py

```diff
@@ -969,11 +1083,19 @@ def _filename(self, package, resource):
         resource = _normalize_path(resource)
         return f"{package_path}/{resource}"
 
-    def _can_implicitly_extern(self, module_name: str):
+    def _check_if_selectively_externed(self, module_name: str) -> bool:
+        for pattern, pattern_info in self.patterns.items():
+            if pattern_info.action == _ModuleProviderAction.SELECTIVE_EXTERN:
+                if pattern.matches(module_name):
+                    return True
+        return False
+
+    def _can_implicitly_extern(self, module_name: str) -> bool:
         top_level_package_name = module_name.partition(".")[0]
-        return top_level_package_name == "torch" or (
-            top_level_package_name not in _DISALLOWED_MODULES
-            and is_stdlib_module(top_level_package_name)
+        if self._check_if_selectively_externed(module_name):
```

Comment:
Is this check necessary? We only implicitly extern standard lib modules now; which are not really candidates for selective externing.

Reply:
Yeah I don't think so, this was a remnant of when torch was externed.